packages/shared/src/dashboards/utilization-card/prometheus-utilization-item.tsx

```diff
@@ -151,4 +163,11 @@ type PrometheusUtilizationItemProps = PrometheusCommonProps & {
   TopConsumerPopover?: React.ComponentType<TopConsumerPopoverProp>;
   setLimitReqState?: (state: LimitRequested) => void;
   basePath?: string;
+  chartType?: 'stacked-area' | 'grouped-line';
+  legendLabel?: string;
+  isBorderhidden?: boolean;
+  customDateTimeFormatter?: (date: Date) => string;
+  showLegend?: boolean;
+  hideCurrentHumanized?: boolean;
```

Comment:
```suggestion
  customizeCurrentValue?: React.ReactNode;
```

pass empty node to <></> to hide current value, basically, the customizeCurrentValue will receive currentValue as arg and another component can customize it.  if customizeCurrentValue is empty then display currentValue as string.